src/test/groovy/myTest.groovy

```diff
@@ -0,0 +1,11 @@
+import spock.lang.Specification
+
+/**
+ * Created by webonise on 14/9/16.
+ */
```

Comment:
Please edit your editors settings. We don't write such a doc strings (whatever it is).